server/routes/changeLocation/handlers/changeLocation.ts

```diff
@@ -11,15 +11,17 @@ export default class ChangeLocationRoutes {
       const prisonCaseloadFromNomis = await this.userService.getPrisonUserCaseloads(user)
       const caseload = prisonCaseloadFromNomis.map(c => ({ value: c.caseLoadId, text: c.description }))
       const checked = req.session.caseloadsSelected
-      const cancelLink = role === AuthRole.CASE_ADMIN ? '/licence/view/cases' : '/licence/approve/cases'
+      const cancelToUrl = req.query.view ? '/licence/view/cases?view=probation' : '/licence/view/cases'
+      const cancelLink = role === AuthRole.CASE_ADMIN ? cancelToUrl : '/licence/approve/cases'
       res.render('pages/changeLocation', { caseload, checked, cancelLink })
```

Comment:
Change location and cancel link are not requested as part of the ticket's AC's. However the change location and cancel link need to be updated as now they serve 2 different views and  following their execution, the user needs to to be returned to their original view.